docs/configuration/settings.rst

```diff
@@ -480,6 +480,28 @@ In order to replace it by another one:
     multiauth.policy.basicauth.use = myproject.authn.BasicAuthPolicy
 
 
+Mohawk request Auth
+:::::::::::::::::::
+
+`Mohawk <https://mohawk.readthedocs.io/en/latest/index.html>`_ implements the 
+`Hawk authorization scheme <https://github.com/hueniverse/hawk>`_, which 
+allows you to authenticate messages from your application backend to Kinto.
```

Comment:
authenticate *requests*.

The rest of the sentence *from your application backend to Kinto.* is not really helpful